packages/indy-vdr/src/pool/DidIdentifier.ts

```diff
@@ -0,0 +1 @@
+export const DID_INDY_REGEX = /^did:indy:((?:[a-z][_a-z0-9-]*)(?::[a-z][_a-z0-9-]*)):([1-9A-HJ-NP-Za-km-z]{21,22})$/
```

Comment:
can you rename this file to `didIdentifier` instead of `DidIdentifier`? We use Pascal case for files that contains classes only. 

See here how to commit a case only name change in git (it's a pain): https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git